app/src/main/java/org/kiwix/kiwixmobile/nav/destination/reader/KiwixReaderFragment.kt

```diff
@@ -263,6 +264,18 @@ class KiwixReaderFragment : CoreReaderFragment() {
     )
   }
 
+  override fun onToggledFullscreen(isFullScreen: Boolean) {
+    if (isFullScreen) {
+      requireActivity().bottom_nav_view.visibility = GONE
```

Comment:
identical to part  of `fun openFullScreen() {`, extract a common method `hideNavBar` or something similar